### DISCUSSION TITLE
后续版本还能用吗？

### DISCUSSION THREAD

**Discussion by twp0** [Z-Siqi/Clash-for-Windows_Chinese#170] [General]
### 你的问题/建议/想说的内容是什么?

```javascript
clash for window 的作者不是跑路了吗？后续的江湖版本是否会被不法分子注入病毒？
正在寻求可替代的代理软件。
var questionMask = `This is a question`;
function saySomething(q) {
return q;
}

``` 

### 版本

[百度一下](www.baidu.com)

**Comment by mcl44** [comment]
cfw和clash core/premium确实删库了，但是停止更新的版本还是能用的(短期内)，代替软件有很多，比如clash-verge-rev和hiddify-next。不过现在很多机场好像喜欢hy协议，cfw不支持。

**Comment by ghost** [comment]
1. clash for window 是开源的, 即使删库了也有别人的 fork, 可以审查 Fndroid(原作者) 最近几次的提交是否有恶意代码
2. 别人二次开发的 clash for window 当然不能用, 除非开源并且有大量的用户
3. 不想用 clash 系列的话, 可以使用 v2ray 系列, Windows 有 v2rayN, Android 有 v2rayNG, 用户数量在 Clash 系列删库之前一直是第二

**Comment by erycon-labs** [comment]
能用，我用了段时间clash-verge，结果发现和远古clash一样的bug，运行久了内核会卡死切换不了模式，又换回来了

**Reply by xbctmri** [reply]
同，并且我直接把cfw换成mihomo内核了。就是直接用不了，得改几行代码然后重新编译。不过也有人开发出了一键脚本。

**Comment by Z-Siqi** [comment]
> 1. clash for window 是开源的, 即使删库了也有别人的 fork, 可以审查 Fndroid(原作者) 最近几次的提交是否有恶意代码
> 2. 别人二次开发的 clash for window 当然不能用, 除非开源并且有大量的用户
> 3. 不想用 clash 系列的话, 可以使用 v2ray 系列, Windows 有 v2rayN, Android 有 v2rayNG, 用户数量在 Clash 系列删库之前一直是第二

指正一下，Clash for Windows 并不完全开源，有代码混淆

**Comment by wowkow** [comment]
> 能用，我用了段时间clash-verge，结果发现和远古clash一样的bug，运行久了内核会卡死切换不了模式，又换回来了

可以分下一下配置地址么，我的配置好像不能用了

**Comment by Aurelius314** [comment]
请问该去哪个网站上找clash for windows 的配置地址呀？之前已经在彩云官网买过套餐的

**Reply by ActorLyle** [reply]
解决了吗？我甚至都不知道这么充值
